internal/expr/query.go

```diff
@@ -1,25 +1,32 @@
 package expr
 
 import (
+	"context"
 	"database/sql"
 	"fmt"
 	"reflect"
 	"strconv"
 	"strings"
 )
 
-func (ce *QueryExpr) QuerySQL() string {
-	return ce.sql
+func (qe *QueryExpr) QuerySQL() string {
+	return qe.sql
 }
 
-func (ce *QueryExpr) QueryArgs() []any {
-	return ce.args
+func (qe *QueryExpr) QueryArgs() []any {
+	return qe.args
+}
+
+func (qe *QueryExpr) QueryContext() context.Context {
+	return qe.ctx
 }
 
 type QueryExpr struct {
 	sql     string
 	args    []any
 	outputs []field
+
+	ctx context.Context
```

Comment:
This seems misplaced here, considering that nothing in this type, or even this package, require the context as it's all expression manipulation.

Reply:
I agree, I've changed this. I had put it here based on what was said in the meeting yesterday.